src/collective/opendata/i18n.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+DOMAIN='collective.opendata'
```

Comment:
we use the script defined here: https://github.com/collective/buildout.plonetest/blob/master/qa.cfg#L56:L84